src/ResourceManager/SiteRecovery/Commands.SiteRecovery.Test/ScenarioTests/SiteRecoveryTests.ps1

```diff
@@ -359,4 +359,47 @@ function Test-SiteRecoveryVaultCRUDTests
 	Remove-AzureRmSiteRecoveryVault -Vault $vaultToBeRemoved
 	$vaults = Get-AzureRmSiteRecoveryVault -ResourceGroupName rg1 -Name v2
 	Assert-True { $vaults.Count -eq 0 }
-}
\ No newline at end of file
+}
+
+
+<#
+.SYNOPSIS
+Site Recovery Fabric Tests New model
+#>
+function Test-SiteRecoveryFabricTest
```

Comment:
>SiteRecoveryFabricTest [](start = 14, length = 22)

Is this good enough test coverage for anyone to contribute to your feature as this is a public repo? I think we need more scenario tests or at least bare minimum unit tests.